src/platform/forms-system/src/js/utilities/ui/index.js

```diff
@@ -75,12 +75,19 @@ export const getFocusableElements = block => {
 export const scrollElementName = 'ScrollElement';
 
 // Set focus on target _after_ the content has been updated
-export function focusOnChange(name, target = '.edit-btn') {
+export function focusOnChange(
+  name,
+  target = '.edit-btn',
+  shadowTarget = undefined,
+) {
   setTimeout(() => {
     const el = $(`[name="${name}${scrollElementName}"]`);
     // nextElementSibling = page form
     const focusTarget = el?.nextElementSibling?.querySelector(target);
-    if (focusTarget) {
+
+    if (focusTarget && shadowTarget) {
+      focusElement(focusTarget.shadowRoot.querySelector(shadowTarget));
```

Comment:
Maybe use optional chaining here in case `shadowRoot` doesn't exist

`focusElement(focusTarget?.shadowRoot?.querySelector(shadowTarget));`

Reply:
Sure, good check